tools/verify-all.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+
+cd "${KUBE_ROOT}"
+
+# exit code, if a script fails we'll set this to 1
+res=0
+
+# run all verify scripts, optionally skipping any of them
+
+if [[ "${VERIFY_GOFMT:-true}" == "true" ]]; then
+  echo "[*] Verifying gofmt..."
+  tools/verify-gofmt.sh || res=1
+  cd "${KUBE_ROOT}"
+fi
+
+if [[ "${VERIFY_GOLINT:-true}" == "true" ]]; then
+  echo "[*] Verifying golint..."
+  tools/verify-golint.sh || res=1
```

Comment:
The list of excluded directories for this seems odd.
1. third_party (Lets not exclude till we have it and need it excluded)
2. vendor (Sounds good)
3. staging/src/k8s.io/client-go/pkg (Lets not exclude till we have it and need it excluded)
4. generated  (Lets not exclude till we have it and need it excluded)
5. clientset_generated  (Lets not exclude till we have it and need it excluded)

Not blocking this change but we should probably follow up on this.

Reply:
Will have a follow up pr on this. Thank you

Reply:
https://github.com/kubernetes/cloud-provider-gcp/pull/218